dist/index.html

```diff
@@ -1,14 +1,78 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Activity Tracker</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="shortcut icon" href="assets/favicon.ico"/>
+
   </head>
   <body>
-    <h1>Activity Tracker</h1>
-    <img src="./images/turing-logo.png" alt="turing logo" height="100px" width="100px">
+    <section class="sidebar">
```

Comment:
ohhhh on this nav bar, would be better element to use instead of <section> , is a <nav> element and also, we need to add calendar, for user to be able to chose a date to check their past stats.  for that we can use <input type = "date">calendar </input>
Thanks